tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -483,8 +483,8 @@ test_i2c_mpr: mpr@18 {
 };
 
 test_i2c_dps310: dps310@48 {
-		compatible = "infineon,dps310";
-		reg = <0x48>;
+	compatible = "infineon,dps310";
+	reg = <0x48>;
```

Comment:
This change is unrelated to the bmp581 driver. Please revert or fix in a separate commit